Ubunu 20.04

Hi !
I didn't managed to install galicaster so i would like to try pyca.

But when installing it , it misses gunicorn3.
Do you have any idea ? I tried to insrall some packages but whithout success..

Futhermore, i would like to try ndi ingest with pyca, is there some docs anywhere ?
Thanks

Hi, 
Thanks for the advices.
I tried with this cmd: python3 -m gunicorn.app.wsgiapp pyca.ui:app
But Pyca failed to launch correctly:

[2020-06-12 13:37:47 +0200] [2245] [INFO] Starting gunicorn 20.0.4
[2020-06-12 13:37:47 +0200] [2245] [INFO] Listening at: http://127.0.0.1:8000 (2245)
[2020-06-12 13:37:47 +0200] [2245] [INFO] Using worker: sync
[2020-06-12 13:37:47 +0200] [2247] [INFO] Booting worker with pid: 2247
[2020-06-12 13:37:47 +0200] [2247] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/gunicorn/arbiter.py", line 583, in spawn_worker
    worker.init_process()
  File "/usr/lib/python3/dist-packages/gunicorn/workers/base.py", line 119, in init_process
    self.load_wsgi()
  File "/usr/lib/python3/dist-packages/gunicorn/workers/base.py", line 144, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/usr/lib/python3/dist-packages/gunicorn/app/base.py", line 67, in wsgi
    self.callable = self.load()
  File "/usr/lib/python3/dist-packages/gunicorn/app/wsgiapp.py", line 49, in load
    return self.load_wsgiapp()
  File "/usr/lib/python3/dist-packages/gunicorn/app/wsgiapp.py", line 39, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/usr/lib/python3/dist-packages/gunicorn/util.py", line 383, in import_app
    mod = importlib.import_module(module)
  File "/usr/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 961, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 973, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'pyca'
[2020-06-12 13:37:47 +0200] [2247] [INFO] Worker exiting (pid: 2247)
[2020-06-12 13:37:47 +0200] [2245] [INFO] Shutting down: Master
[2020-06-12 13:37:47 +0200] [2245] [INFO] Reason: Worker failed to boot.

--- a/init/systemd/pyca-ui.service
+++ b/init/systemd/pyca-ui.service
@@ -8,7 +8,7 @@ After=pyca.service
 [Service]
 Type=simple
 User=pyca
-ExecStart=/usr/bin/gunicorn --config=/etc/pyca/gunicorn.conf.py pyca.ui:app
+ExecStart=/usr/bin/python3 -m gunicorn.app.wsgiapp --config=/etc/pyca/gunicorn.conf.py pyca.ui:app
 Restart=always
 RestartSec=10
 

